src/main/resources/alma/fix/macros.fix

```diff
@@ -252,6 +252,40 @@ do put_macro("provenanceLinks")
   append("$[field]", "#!")
 end
 
+# opacLink
+
+do put_macro("opacLink")
+  if exists("zdbId")
+    copy_field("$i.$[field]", "$i.@opacLinkZdbId")
+    lookup("$i.@opacLinkZdbId","isil2opac_zdbId", delete:"true")
+    split_field("$i.@opacLinkZdbId","\\{zdbid\\}")
+  elsif exists("hbzId")
+    copy_field("$i.$[field]", "$i.@opacLinkHbzId")
+    lookup("$i.@opacLinkHbzId","isil2opac_hbzId", delete:"true")
+    split_field("$i.@opacLinkHbzId","\\{hbzid\\}")
+  end
+  if exists("issn[].1")
+    copy_field("$i.$[field]", "$i.@opacLinkIssn")
+    lookup("$i.@opacLinkIssn","isil2opac_issn", delete:"true")
+    split_field("$i.@opacLinkIssn","\\{issn\\}")
+  end
+  if exists("isbn[].1")
+    copy_field("$i.$[field]", "$i.@opacLinkIsbn")
+    lookup("$i.@opacLinkIsbn","isil2opac_isbn", delete:"true")
+    split_field("$i.@opacLinkZdbId","\\{isbn\\}")
+  end
+  set_array("hasItem[].$last.seeAlso[]")
```

Comment:
Does this mean that there can be multiple links in the future?

Reply:
nope, but the property is defined as set in the context.

Reply:
Okay, I'll take your word for it ;)

Reply:
Nope, the context says it has to be an array, since it is used elsewhere in the record too.

Reply:
I understand that. I meant that I trust that you don't sneak additional URLs in there in the future.